### PR TITLE
More tweaks

### PR DESCRIPTION
**Pull request by illicitonion**

### CODE REVIEW

File: Big-Spender/readme.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ INSERT YOUR QUERY HERE
 
 **Claire:** I think that's all the same to me.
 
+**You:** What about "fees"?
+
+**Claire:** Yes, we want to include "fees" as well as "fee"... But definitely not "coffee"!
+
+(Hint: For this particular data-set, you don't need to do anything very complicated here, just using a `LIKE` operator. In the future, if you need to be more sophisticated, you may want to read about [stemming](https://www.compose.com/articles/indexing-for-full-text-search-in-postgresql/) and the `to_tsvector` and `to_tsquery` functions in Postgres).
```
Comment (SallyMcGrath) [inline Big-Spender/readme.md:68]: I don't LOVE inserting notes like this but won't block.

**Review by SallyMcGrath — APPROVED** [review]
LGTM TY

**Review by SallyMcGrath — APPROVED** [review]
LGTM TY

**Comment by SallyMcGrath** [comment]
OH PS I had already opened a PR with similar changes, and Keith reviewed. Your changes seem fine to me, but Keith's note could go in: https://github.com/CodeYourFuture/Module-Databases/pull/19#discussion_r1167932547

**Comment by illicitonion** [comment]
> OH PS I had already opened a PR with similar changes, and Keith reviewed. Your changes seem fine to me, but Keith's note could go in: [#19 (comment)](https://github.com/CodeYourFuture/Module-Databases/pull/19#discussion_r1167932547)

Let's run with your one, I can find a better way to add a prompt for better stemming separately.

**Pull request closed without merging**